isaric/parsers/ccp-guinea.toml

```diff
@@ -578,43 +560,38 @@
 [visit.treatment_antivirals]
   combinedType = "any"
   fields = [
-    { field = "other_ant_viral_prmd", ref = "Y/N/NK" },
     { field = "daily_antiviral_cmyn", values = { 1 = true, 0 = false } },
   ]
 
 [visit.treatment_high_flow_nasal_cannula]
-  field = "overall_oxygen_cmtype___2"
-  description = "Oxygen interface type - HF nasal cannula"
-  values = { 2 = true }
+  combinedType = "any"
+  fields = [
+    { field = "daily_oxygen_cmtype_1", values = { 2 = true } },
+    { field = "can_oxy_ther", values = { 1 = true, 2 = false } },
+  ]
 
 [visit.treatment_dialysis]
   combinedType = "any"
 
-  [[visit.treatment_dialysis.fields]]
-    field = "dialysis_disch"
-    description = "Dialysis/kidney treatment"
-    values = { 1 = true, 0 = false }
-
   [[visit.treatment_dialysis.fields]]
     field = "dialy_hemo"
     description = "Dialysis/Hemofiltration?"
-    ref = "Y/N/NK"
+    ref = "Y/N"
 
 [visit.treatment_ecmo]
-  field = "overall_extracorp_prtrt"
+  field = "ecls"
   description = "Extracorporeal (ECMO) support"
-  ref = "Y/N/NK"
+  ref = "Y/N"
 
 [visit.treatment_corticosteroid]
   field = "daily_corticost_cmyn"
   description = "Corticosteroids"
-  ref = "Y/N/NK"
+  ref = "Y/N"
 
 [visit.treatment_corticosteroid_type]
   combinedType = "set"
   excludeWhen = "none"
   fields = [
-    { field = "dexamethasone", values = { 1 = "Dexamthasone" } },
     { field = "hydrocortisone", values = { 1 = "Hydrocortisone" } },
```

Comment:
@pipliggins I'd noted from my initial review to use daily_corticost_cmtrt___1 = 1 to map to the visit attribute visit.treatment_dexamethasone. Maybe this now comes under this corticosteroid_type attribute? If so, daily_corticost_cmtrt___1 (dexa), ___2 (hydrocortisone), ___3 (prednisolone), and ___4 (methylprednisolone) can be mapped here also